04-onion-routing.md

```diff
@@ -150,52 +155,75 @@ The overall structure of the packet is as follows:
 2. data:
    * [`1`:`version`]
    * [`33`:`public_key`]
-   * [`20*65`:`hops_data`]
+   * [`20*FRAME_SIZE`:`hops_data`]
```

Comment:
Perhaps we should also lift `20` into a `MAX_HOPS` constant while we're at it?

Reply:
Since *FRAME_SIZE* (65 bytes) is fixed and packet length is fixed too (1300 bytes), I suggest *FRAME_COUNT* (max doesn't really make sense because it implies we could use less, which would result in a smaller onion which isn't allowed).